buildspecs/osx_x86-64.spec

```diff
@@ -0,0 +1,282 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Copyright (c) 2006, 2017 IBM Corp. and others
```

Comment:
Should this date be `2017, 2017`? Or is it a modified version of the linux spec (which would justify the 2006 date)?

Reply:
Its a modified version of the linux_x86-64.spec.